apps/docs/src/app/platform/component-docs/platform-list/platform-list-examples/platform-list-with-buttons-example.component.html

```diff
@@ -0,0 +1,14 @@
+<fdp-list>
+    <fdp-standard-list-item [titleIcon]="'address-book'" title="Title1" [secondayIcons]="[{'icon':'accept'
```

Comment:
![2020-07-09_19-09-16 (1)](https://user-images.githubusercontent.com/53509521/87047467-2ee53b80-c218-11ea-8284-0b365d4c2457.gif)
The selected state is not applying properly on the icons when the other icon is clicked.

Reply:
sure, will look into it

Reply:
As per Fiori3, it is the expected behavior of Styles.
<img width="1300" alt="Screen Shot 2020-07-09 at 10 56 50 PM" src="https://user-images.githubusercontent.com/53487468/87071254-952d8680-c237-11ea-8e30-9ea2697a6d07.png">

Reply:
Fiori3 is only talking about white text in active state. In the gif i shared, I am trying to show that when one icon is clicked, the other icon's active state is not applied properly. How is it in UI5 samples? I couldn't find a sample showing two icons at the right.